### DISCUSSION TITLE
vue-i18n registers an alias not starting with a special char, leading to a warning

### DISCUSSION THREAD

**Discussion by lourot** [vikejs/vike#1356] [Discussion]
(Asked by @ulfgebhardt on Discord)

Reproducer: https://github.com/IT4Change/boilerplate-frontend/

```
[vike][Warning] vite.config.ts defines an invalid resolve.alias which will be deprecated in the next major release: a path alias needs to start with a special character, see https://vike.dev/path-aliases#vite
```

The project itself defines only aliases starting with `#` but it seems that a dependency vue-i18n is creating one that doesn't:

![image](https://github.com/vikejs/vike/assets/11795312/4bbab6a8-53f9-49c6-a605-89e53dfd5e78)

Not sure how to go about that :thinking:

**Comment by brillout** [comment]
See https://github.com/vikejs/vike/issues/1146.
